src/helpers/bundler.ts

```diff
@@ -0,0 +1,15 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
+
+export function bundlify(resources: Array<R4.IResourceList>): R4.IBundle {
+  return {
+    resourceType: 'Bundle',
+    type: R4.BundleTypeKind._transaction,
```

Comment:
We may want to support a general bundle that is not of type transaction.   Something more along the lines of collection or something like that were we just have everything packaged together.

Reply:
Yeah I initially was going to do that. The only reason I did txn was that I wasn't sure if we planned to actually post this valuesets anywhere? I guess not if we have all the resources. The only HTTP requests would be for ones that are on some terminology server rather than defined by the IG ppl

Reply:
pushed